### DISCUSSION TITLE
Can I list my own repositories at comandline?

### DISCUSSION THREAD

**Discussion by yousan** [community/community#22633] [New to GitHub]
<p>Hi.</p>
<p>I want to clone my repositries at commandline.</p>
<p>Can I list my own repositories at comandline?</p>
<p>Such as</p>
<p>    git list | grep some-my-repo</p>
<p>I am assuming that <code>curl api.github.com</code> or <code>hub</code>.</p>
<p>Thanks</p>

**Comment by mpboom** [comment, marked as answer]
<p>Check out the documentation here:</p>
<p><a href="https://developer.github.com/v3/repos/#list-your-repositories" rel="nofollow noopener">https://developer.github.com/v3/repos/#list-your-repositories</a></p>
